angular.json

```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
   "cli": {
-    "analytics": "b0fec727-d21d-49bf-b321-5c65a26c8df5"
+    "analytics": "b0fec727-d21d-49bf-b321-5c65a26c8df5",
```

Comment:
didn't know this even existed. Do we want to disable it? Especially since this is a fork...

Reply:
Yes, and done.